crates/stateless/Cargo.toml

```diff
@@ -0,0 +1,42 @@
+[package]
```

Comment:
you'd want to make this crate `no-std` compatible if it's intended to be run on risc v